test/fixtures/rules/std_rule_cases/L026.yml

```diff
@@ -39,4 +45,38 @@ test_pass_object_referenced_4:
     FROM `example_dataset2.example_table2` AS et2
   configs:
     core:
-      dialect: bigquery
\ No newline at end of file
+      dialect: bigquery
+    rules:
+      L026:
+        force_enable: True
+
+test_pass_object_referenced_5:
```

Comment:
Since tests 5-7 are closely related, should we indicate that somehow through the names, e.g.
* test_pass_object_referenced_5_a
* test_pass_object_referenced_5_b
* test_pass_object_referenced_5_c